test/tests/parallel/set2/test_helper.bats

```diff
@@ -130,43 +140,55 @@ load 'test_helper/common'
   assert_failure
 }
 
-@test "container_has_service_running/wait_for_service" {
+@test "${TEST_NAME_PREFIX} container_has_service_running/wait_for_service" {
+  # the container name is used here explicitly, because it cannot be a local
+  # variable as the `teardown` function would not have the correct scope
+  # and does not see the variable
+
   local PRIVATE_CONFIG
-  PRIVATE_CONFIG=$(duplicate_config_for_container .)
+  PRIVATE_CONFIG=$(duplicate_config_for_container . dms-test-services_running)
 
   # variable not local to make visible to teardown
-  CONTAINER_NAME=$(docker run -d --rm \
+  docker run -d --rm --name dms-test-services_running \
     -v "${PRIVATE_CONFIG}":/tmp/docker-mailserver \
     -h mail.my-domain.com \
-    -t "${NAME}")
+    -e ENABLE_CLAMAV=0 \
+    -e ENABLE_AMAVIS=0 \
+    -e ENABLE_FAIL2BAN=0 \
+    -e ENABLE_SPAMASSASIN=0 \
```

Comment:
These ENV were added for what reason?

Reply:
Reverted those for which the default is already 0